mass-deployer/README.md

```diff
@@ -54,11 +57,25 @@ You can use this [example](./example/conf.yaml) for further guidance,
 tfrobot deploy -c path/to/your/config.yaml
 ```
 
-6.  Run the canceler with path to the config file
+## Usage
+### Subcommands:
+
+-   **deploy:** used to mass deploy groups of vms with specific configurations
+```bash
+tfrobot deploy -c path/to/your/config.yaml
+```
+
+-   **cancel:** used to cancel all vms deployed using specific configurations
 ```bash
 tfrobot cancel -c path/to/your/config.yaml
 ```
 
+### Flags:
+| Flag | Usage |
+| :---:   | :---: |
+| -c | used to specify path to configuration file |
+| -c | used to specify path to configuration file |
```

Comment:
-o is not added. and should be specified that format will be based on file extension for both cases